lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -1035,6 +1035,23 @@
     <!-- rdeval types -->
     <datatype extension="rd" type="galaxy.datatypes.binary:Binary" mimetype="application/octet-stream" display_in_upload="true" subclass="true" description="Rdeval read sketch"/>
   </registration>
+
+  <visualization_mappings>
+    <!-- 
+      This section maps datatypes to their preferred visualization plugins.
+      These visualizations will be used as the default preview when viewing datasets of the specified type.
+      
+      Attributes:
+      - datatype: The file extension/datatype to map
+      - visualization: The visualization plugin to use
+      - default_params: Optional JSON-formatted parameters to pass to the visualization
+    -->
+    <visualization_mapping datatype="h5" visualization="vitessce" />
```

Comment:
Didn't you want to use h5web for h5 ?

Reply:
Yes, and I think I want to restructure this to shift the mappings up into the individual datatypes, too -- incoming...